packages/core-components/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -80,11 +80,11 @@ export function ErrorPage(props: IErrorPageProps) {
           Looks like someone dropped the mic!
         </Typography>
         <Typography variant="h6">
-          <Link data-testid="go-back-link" onClick={() => navigate(-1)}>
+          <Link to=".." data-testid="go-back-link">
```

Comment:
Switching from a JavaScript-based browser history back to just a path is likely not going to work for this situation? Others may have more technical suggestions on how to fix, but thing this needs some tweaks.

Reply:
I used this ".." to go back because I noticed this is already used in other components. So I kept the pattern.
You can find one example at backstage\plugins\circleci\src\components\BuildWithStepsPage\BuildWithStepsPage.tsx

Reply:
@adamdmharvey do I adjust this or follow the pattern?

Reply:
not sure; will need to leave to others or the core maintainers to check on this :) if you'd tested it manually, i'm good with it!